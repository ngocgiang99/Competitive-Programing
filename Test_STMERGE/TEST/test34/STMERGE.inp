4
3 2
990916890 685978634 
646117153 446094787 
285588134 546829356 
6 4
636964617 76731255 220902975 862030487 
152435367 430183394 39567765 221428672 
339596838 103232928 624970166 174001264 
615080042 458254631 883751847 385406268 
228861593 457117996 437855918 779217144 
418079768 8242852 385433166 284084409 
1 4
679094350 101012436 602512344 41219328 
3 3
673227812 750812431 731308478 
41368751 140391379 610291940 
403078137 564861016 107897227 
