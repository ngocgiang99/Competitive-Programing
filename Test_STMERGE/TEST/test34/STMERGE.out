285588134
290300284
41219328
149265978
